cli/cmd/auth.go

```diff
@@ -87,11 +111,27 @@ More precisely, the Keptn CLI stores the endpoint and API token using *pass* in
 
 func init() {
 	rootCmd.AddCommand(authCmd)
+	authParams = &authCmdParams{}
 
-	endPoint = authCmd.Flags().StringP("endpoint", "e", "", "The endpoint exposed by the Keptn installation (e.g., api.keptn.127.0.0.1.xip.io)")
-	authCmd.MarkFlagRequired("endpoint")
-	apiToken = authCmd.Flags().StringP("api-token", "a", "", "The API token to communicate with the Keptn installation")
-	authCmd.MarkFlagRequired("api-token")
+	authParams.endPoint = authCmd.Flags().StringP("endpoint", "e", "", "The endpoint exposed by the Keptn installation (e.g., api.keptn.127.0.0.1.xip.io)")
+	authParams.apiToken = authCmd.Flags().StringP("api-token", "a", "", "The API token to communicate with the Keptn installation")
+	authParams.exportConfig = authCmd.Flags().BoolP("export", "c", false, "To export the current cluster config i.e API token and Endpoint")
+}
+
+func verifyAuthParams(authParams *authCmdParams) error {
+
+	if *authParams.exportConfig {
+		return nil
+	}
+
+	if authParams.endPoint == nil && authParams.apiToken == nil {
+		return errors.New("required flag(s) \"api-token\", \"endpoint\" not set")
+	} else if authParams.endPoint == nil {
+		return errors.New("required flag \"endpoint\" not set")
+	} else if authParams.apiToken == nil {
+		return errors.New("required flag \"api-token\" not set")
```

Comment:
Actually, there is a little problem here, you're comparing to `nil`, but the default value (defined above) is `""`

Reply:
Yes, that's because while mocking credentialstore, we are passing `""` for endpoint and api-token hence, my tests were getting failed before. I am checking `nil` so I can validate where the flag is passed or not.

Reply:
@christian-kreuzberger-dtx Have made the changes, please check, Thanks :)

Reply:
I only mentioned it becaise I ran into this problem when run `keptn auth` without params:

```
$ ./keptn auth
Starting to authenticate
Failed to resolve hostname 
Retrying...
```

And this is what I got with 0.7.1:
```
$ keptn auth
Error: required flag(s) "api-token", "endpoint" not set
```

Reply:
Yes, fixed this